[Bug]: `@automattic/format-currency` has an undeclared dependency on `tslib`

### Quick summary

`@automattic/format-currency` has an undeclared dependency on `tslib`

### Steps to reproduce

1. Create a new project with this `package.json`
   ```json
   {
       "type": "module",
       "dependencies": {
           "@automattic/format-currency": "^1.0.0",
           "react": "^17.0.2"
       }
   }
   ```
2. Create this `index.js`
   ```js
   import fc from '@automattic/format-currency';
   
   const formatCurrency = fc.default || fc;
   
   console.log( formatCurrency( 1.23, 'USD' ) );
   ```
3. Do `npm install` or whatever.
4. Run it as `node index.js`


### What you expected to happen

```
$1.23
```

### What actually happened

```
node:internal/modules/cjs/loader:933
  const err = new Error(message);
              ^

Error: Cannot find module 'tslib'
Require stack:
- /tmp/x/xx/node_modules/.pnpm/@automattic+format-currency@1.0.0_react@17.0.2/node_modules/@automattic/format-currency/dist/cjs/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/tmp/x/xx/node_modules/.pnpm/@automattic+format-currency@1.0.0_react@17.0.2/node_modules/@automattic/format-currency/dist/cjs/index.js:4:17)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:168:29) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/tmp/x/xx/node_modules/.pnpm/@automattic+format-currency@1.0.0_react@17.0.2/node_modules/@automattic/format-currency/dist/cjs/index.js'
  ]
}
```

### Context

_No response_

### Browser

_No response_

### Simple/Atomic

_No response_

### Other notes

_No response_

### Reproducibility

Consistent

### Severity

_No response_

### Available workarounds?

_No response_

### Workaround details

_No response_

--- a/packages/format-currency/CHANGELOG.md
+++ b/packages/format-currency/CHANGELOG.md
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 1.0.0 - 2022-04-25
 ### Fixed
 - Fixed tslib dependency
 

--- a/packages/format-currency/CHANGELOG.md
+++ b/packages/format-currency/CHANGELOG.md
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- Fixed tslib dependency
+
 ## 1.0.0 - 2022-04-20
 ### Added
 - Initial release

--- a/packages/format-currency/package.json
+++ b/packages/format-currency/package.json
@@ -29,12 +29,11 @@
 		"prepack": "yarn run clean && yarn run build",
 		"watch": "tsc --build ./tsconfig.json --watch"
 	},
-	"peerDependencies": {
-		"react": "^17.0.2"
+	"dependencies": {
+		"tslib": "^2.3.0"
 	},
 	"devDependencies": {
 		"@automattic/calypso-typescript-config": "workspace:^",
-		"tslib": "^2.3.0",
 		"typescript": "^4.5.5"
 	},
 	"files": [

--- a/packages/format-currency/package.json
+++ b/packages/format-currency/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/format-currency",
-	"version": "1.0.0",
+	"version": "1.0.1",
 	"description": "JavaScript library for formatting currency.",
 	"main": "dist/cjs/index.js",
 	"module": "dist/esm/index.js",

